src/components/ThreejsEditorModal.js

```diff
@@ -27,9 +27,8 @@ export class ThreejsEditorModal extends ModalDialog {
     }
 
     setNumberFormat() {
-        Number.prototype.format = function () {
-            return this.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,");
-        };
+        /*eslint no-extend-native: [0, { "exceptions": ["Object"] }]*/
```

Comment:
Hard to understand what this function does and why it's necessary. Docstring with detailed explanation?

Reply:
Still - What is Eslint error? Why does it appear and why do we try to modify a read-only object?

Reply:
As the comment says the function is used inside three.js editor codebase to format the numbers. The editor does not start without it. The ESLint line is a way to turn off the warning shown in console.

Reply:
Lets put your last comment on line 30

Reply:
Done.